selfdrive/controls/controlsd.py

```diff
@@ -578,6 +578,13 @@ def state_control(self, CS):
     sr = max(params.steerRatio, 0.1)
     self.VM.update_params(x, sr)
 
+    # Update Torque Params
+    if self.CP.lateralTuning.which() == 'torque':
+      torque_params = self.sm['liveTorqueParameters']
+      # Todo: Figure out why this is needed, and remove it
+      if (torque_params.latAccelFactorFiltered > 0) and (self.sm.valid['liveTorqueParameters']) and (torque_params.useParams):
```

Comment:
what's the check on latAccelFactorFiltered for? the msg should have a flag for validity
```suggestion
      if (torque_params.latAccelFactorFiltered > 0) and self.sm.all_checks(['liveTorqueParameters']) and torque_params.useParams:
```